packages/testing/jest.config.js

```diff
@@ -0,0 +1,7 @@
+/** @type {import('@jest/types').Config.InitialOptions} */
+const config = {
+  rootDir: './',
+  setupFilesAfterEnv: ['<rootDir>/jest.setup.ts'],
```

Comment:
I didn't realize that these could be untranspiled. Does jest use babel-jest on these?

Reply:
It does! I'm not sure exactly when that started happening, but I've been doing the same type of setup for the last year or so? There is a note is in a [comment here](https://jestjs.io/docs/getting-started#using-babel).

> Note: babel-jest is automatically installed when installing Jest and will automatically transform files if a babel configuration exists in your project. To avoid this behavior, you can explicitly reset the transform configuration option:

Reply:
Very cool!